tools/StaticAnalysis/ExampleAnalyzer/utils.ps1

```diff
@@ -564,7 +564,7 @@ function Get-NonExceptionRecord{
     $errors = @()
     foreach($record in $records){
         $needAdd = $true
-        $exceptionPath = Join-Path -Path $exceptionPaths -ChildPath "Az.$($record.Module)" -AdditionalChildPath "ExampleIssues.csv"
+        $exceptionPath = Join-Path -Path $exceptionPaths -ChildPath (EnsureModuleName($record.Module)) -AdditionalChildPath "ExampleIssues.csv"
```

Comment:
Previously `-ChildPath` will be "Az.Az.***". That's why suppression did not work.